compiler/src/dotty/tools/dotc/transform/Bridges.scala

```diff
@@ -56,7 +56,13 @@ class Bridges(root: ClassSymbol)(implicit ctx: Context) {
       bridgesScope.lookupAll(member.name).exists(bridge =>
         bridgeTarget(bridge) == member && bridge.signature == other.signature)
     def info(sym: Symbol)(implicit ctx: Context) = sym.info
-    def desc(sym: Symbol)= i"$sym${info(sym)(preErasureCtx)} in ${sym.owner}"
+    def desc(sym: Symbol)= {
```

Comment:
Maybe:
```scala
val infoStr = sym.showDcl(preErasureCtx)
i"$infoStr in ${sym.owner}"
```
This will print `def produce: => X`. But I think we could configure the printer to not print the `=>`. Other part of the compiler (such as error messages) could benefit from it.

Reply:
The problem is that printing the type will always print the `=>`. Otherwise it would be confusing. I would should work on the Printer but it has to look at the larger context to do the right thing.